File: Android/PersonitaSqlite/app/src/main/java/com/example/personitasqlite/Personita.java
package com.example.personitasqlite;

public class Personita {
    private String nombre;

    public Personita() {

    }

    public Personita(String nombre) {
        this.nombre = nombre;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    @Override
    public String toString() {
        return nombre;
    }

}

File: Android/personaSQLite/app/src/main/java/com/example/personotasqlite/Personita.kt
package com.example.personotasqlite

class Personita {
    var nombre: String? = null
    var apPat: String? = null
    var apMat: String? = null
    var nacimiento: String? = null
    var sexo: String? = null

    constructor()

    constructor(nombre: String?, apPat: String?, apMat: String?, nacimiento: String?, sexo: String?) {
        this.nombre = nombre
        this.apPat = apPat
        this.apMat = apMat
        this.nacimiento = nacimiento
        this.sexo = sexo
    }


    override fun toString(): String {
        return "Nombre: ${nombre ?: ""}, Apellido Paterno: ${apPat ?: ""}, Apellido Materno: ${apMat ?: ""}, Fecha de Nacimiento: ${nacimiento ?: ""}, Sexo: ${sexo ?: ""}"
    }
}